test/e2e/create_test.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
You make this change in this commit and then revert in the next. Please fix.

Reply:
My IDE loves to add this in when auto-formatting code :smile: